src/components/Answers.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+function Answers(props) {
+  const { answers, forCorrect } = props;
+  const getFilteredAnswers = answers.filter((answer) => (
```

Comment:
Variable names should have meanings. It should indicate what you are filtering on the basis of.

Reply:
Done